conanfile.py

```diff
@@ -11,26 +11,34 @@ class RagelConan(ConanFile):
                   "Ragel targets C, C++ and ASM. "
     homepage = "http://www.colm.net/open-source/ragel"
 
-    settings = {"os": ["Windows"],
+    settings = {"os": ["Windows", "Linux", "Macos"],

```

Comment:
if it is an installer package, may be just os_build and arch_build are enough? as well as just one CI configuration per OS.

Reply:
Good point, I should checked other installers before to do this.

Reply:
I've changed to os_build/arch_build